src/components/CustomerDetails/InputNricSection.tsx

```diff
@@ -73,7 +73,7 @@ export const InputNricSection: FunctionComponent<InputNricSection> = ({
       <View style={styles.inputAndButtonWrapper}>
         <View style={styles.inputWrapper}>
           <InputWithLabel
-            label="Enter NRIC number"
+            label="Enter identification number"
```

Comment:
It's wrapping on my phone rather than being on a row on its own, can check if it is low effort to fix?

Instead of 

```
Enter identification number
[ Input             ] [ Check ]
```

it is

```
Enter identification 
number
[ Input             ] [ Check ]
```

Reply:
seems like the view container for label is rendered with the input, so the label wont exceed till its above the button, might take a bit more effort to change, shall we revisit this? will also check with Eida on her view too!

Reply:
I see, ok to revisit but if this is something we want to fix in the future can open a Trello card for it?